docker/config/default.config.envsubst

```diff
@@ -24,6 +24,7 @@ partition.fragments=$RADIXDLT_PARTITION_FRAGMENTS
 universe.witness=$RADIXDLT_UNIVERSE_WITNESSES
 debug.profiler=$RADIXDLT_DEBUG_PROFILER
 consensus.fixed_quorum_size=$RADIXDLT_CONSENSUS_FIXED_QUORUM_SIZE
+consensus.pacemaker_timeout_millis=$RADIXDLT_PACEMAKER_TIMEOUT_MILLIS
```

Comment:
I think you will need to set this environment variable in (at least) Dockerfile as well, otherwise we may end up with a timeout of 0.

Reply:
Have setup default value of 5000ms in Dockerfile